front/src/applications/editor/tools/switchEdition/components.tsx

```diff
@@ -8,32 +8,36 @@ import { featureCollection, point } from '@turf/helpers';
 import nearestPoint from '@turf/nearest-point';
 import { Position } from 'geojson';
 
-import EditorContext from '../../context';
-import { EntityObjectOperationResult, SwitchEntity, TrackSectionEntity } from '../../../../types';
-import colors from '../../../../common/Map/Consts/colors';
-import GeoJSONs from '../../../../common/Map/Layers/GeoJSONs';
-import { SwitchEditionState } from './types';
-import EditorForm from '../../components/EditorForm';
-import { save } from '../../../../reducers/editor';
-import { FlatSwitchEntity, flatSwitchToSwitch, getNewSwitch, isSwitchValid } from './utils';
+import EditorForm from 'applications/editor//components/EditorForm';
+import EntitySumUp from 'applications/editor/components/EntitySumUp';
+import EntityError from 'applications/editor/components/EntityError';
+import EditorContext from 'applications/editor/context';
+import { getEntity } from 'applications/editor/data/api';
+import { flattenEntity } from 'applications/editor/data/utils';
+import { ExtendedEditorContextType } from 'applications/editor/tools/editorContextTypes';
 import {
-  getSwitchesLayerProps,
-  getSwitchesNameLayerProps,
-} from '../../../../common/Map/Layers/Switches';
-import { flattenEntity } from '../../data/utils';
-import EntitySumUp from '../../components/EntitySumUp';
-import EntityError from '../../components/EntityError';
-import { getEntity } from '../../data/api';
-import { getInfraID } from '../../../../reducers/osrdconf/selectors';
-import { getMap } from '../../../../reducers/map/selectors';
-import { ExtendedEditorContextType } from '../editorContextTypes';
-import { CustomSchemaField } from './components/CustomSchemaField';
-import useSwitch from './useSwitch';
+  FlatSwitchEntity,
+  flatSwitchToSwitch,
+  getNewSwitch,
+  isSwitchValid,
+} from 'applications/editor/tools/switchEdition/utils';
+import { SwitchEditionState } from 'applications/editor/tools/switchEdition/types';
```

Comment:
This one too can stay relative, it’s the same component in the same folder

Reply:
Done